superduperdb/components/model.py

```diff
@@ -41,6 +44,31 @@ def __call__(self, X):
         return self.callable(X, **self.kwargs)
 
 
+class Inputs:
+    def __init__(self, fn, predict_kwargs: t.Dict = {}):
+        sig = inspect.signature(fn)
+        sig_keys = list(sig.parameters.keys())
+        params = []
+        for k in sig_keys:
+            if k in predict_kwargs or (k == 'kwargs' and sig.parameters[k].kind == 4):
```

Comment:
How about the `*args`?

Reply:
Hey so this logic is to skip params of the object which matches `self.predict_kwargs` since those will be set by the predict_kwargs :)

Reply:
OK, Great